velox/expression/tests/ExprOptimizerTest.cpp

```diff
@@ -323,8 +286,8 @@ TEST_F(ExprOptimizerTest, makeFailExpr) {
   // Primitive types.
   assertPrestoFailExpr("0 / 0", prestoFailCall(), ROW({}));
   assertPrestoFailExpr(
-      "if(a = 2 * 2, a / abs(-1 * 3), 0 / 0)",
-      fmt::format("if(a = 4, a / 3, {})", prestoFailCall()),
+      "if(false, a * abs(-1 * 3), 0 / 0)",
```

Comment:
similarly, over here, lets remove this since its not adding any test coverage unless switch rewrite is added